pgml-dashboard/src/api/deployment/mod.rs

```diff
@@ -0,0 +1,67 @@
+use rocket::route::Route;
+use sailfish::TemplateOnce;
+
+use crate::{
+    guards::ConnectedCluster,
+    responses::{Error, ResponseOk},
+};
+
+use crate::templates::{components::NavLink, *};
+
+use crate::models;
+use crate::templates;
+use crate::utils::tabs;
+use crate::utils::urls;
+
+use std::collections::HashMap;
+
+pub mod deployment_models;
+pub mod notebooks;
+pub mod projects;
+pub mod snapshots;
+pub mod uploader;
+
+#[get("/deployments")]
```

Comment:
This conflicts with our cloud route.

Reply:
This is not new code, I just moved all the deployment/engine stuff out of lib.rs and into an organized file structure.  We add /dashboard to the front.  So the actual url is /dashboard/deployments so to be /engine/deployments.

Reply:
Right, sorry. Been a while since I looked at this code.